src/main/java/org/alliancegenome/curation_api/model/ingest/json/dto/ExperimentalConditionDTO.java

```diff
@@ -13,6 +14,8 @@
     private String conditionQuantity;
     private String anatomicalOntologyId;
     private String geneOntologyId;
+
+    @JsonProperty("NCBITaxonId")
```

Comment:
This was added because I kept running into an issue while loading ZFIN DAF files. I would get a message "Unrecognized field "NCBITaxonId"; (class org.alliancegenome.curation_api.model.ingest.json.dto.ExperimentalConditionDTO), not marked as ignorable". This seemed to fix it.

Reply:
This is only an issue with the Zfin file. We have a new Zfin file where we won't need this... I would suggest not added it.

Reply:
I should give you the most recent file, Adam.